libs/s25main/desktops/dskGameLobby.cpp

```diff
@@ -222,10 +244,9 @@ dskGameLobby::dskGameLobby(ServerType serverType, std::shared_ptr<GameLobby> gam
             combo->AddString(helpers::format(_("Tournament: %u minutes"), duration));
     }
 
-    // "Geschwindigkeit"
-    AddText(33, DrawPoint(400, 315), _("Speed:"), COLOR_YELLOW, FontStyle{}, NormalFont);
-    combo =
-      AddComboBox(43, DrawPoint(600, 310), Extent(180, 20), TextureColor::Grey, NormalFont, 100, !gameLobby_->isHost());
+    AddText(ID_txtSpeed, DrawPoint(400, 315), _("Speed:"), COLOR_YELLOW, FontStyle{}, NormalFont);
+    combo = AddComboBox(ID_cbSpeed, DrawPoint(600, 310), Extent(180, 20), TextureColor::Grey, NormalFont, 100,
+                        !gameLobby_->isHost());
     combo->AddString(_("Very slow")); // Sehr Langsam
```

Comment:
BTW: Feel free to remove such comments that literally state the same as the code while modifying this or very adjacent code.

Reply:
I tried to do so, but some must've slipped through the fingers. Will do another pass through that.